tests/components/zwave_js/test_config_flow.py

```diff
@@ -3513,3 +3519,52 @@ async def mock_backup_nvm_raw():
 
     assert result["type"] == FlowResultType.ABORT
     assert result["reason"] == "restore_failed"
+
+
+async def test_get_driver_failure(hass: HomeAssistant, integration, client) -> None:
+    """Test get driver failure."""
+
+    handler = OptionsFlowHandler()
+    handler.hass = hass
+    handler._config_entry = integration
+    client.driver = None
+
+    with pytest.raises(data_entry_flow.AbortFlow):
+        await handler._get_driver()
+
+
+async def test_hard_reset_failure(hass: HomeAssistant, integration, client) -> None:
+    """Test hard reset failure."""
+    handler = OptionsFlowHandler()
+    handler.hass = hass
+    handler._config_entry = integration
+
+    client.driver.async_hard_reset = AsyncMock(
+        side_effect=FailedCommand("test_error", "unknown_error")
+    )
+
+    result = await handler.async_step_instruct_unplug()
```

Comment:
We shouldn't test the step methods directly. Helpers are ok to unit test but the steps should be tested by starting the flow and then completing the flow to a create entry or abort result.